content/collections/create.en.md

```diff
@@ -312,7 +312,7 @@ On your local machine:
 
 #### Create the SSH key
 
-- Create a new SSH key: `ssh-keygen -t ed25519 -C bot@opentermsarchive.org -P "" -f ./<collection_name>-key`
+- Create a new SSH key: `ssh-keygen -t ed25519 -C bot@opentermsarchive.org -P "" -f ~/.ssh/<collection_name>-key`
```

Comment:
Why creating this key in your ssh folder whereas it should be deleted from your computer after backing it up and attached to the OTA-Bot user?